File: cloud-collect-core/src/main/java/com/windea/demo/cloudcollect/core/exception/NotImplementedException.java
package com.windea.demo.cloudcollect.core.exception;

/**
 * 功能未实现的异常。
 */
public class NotImplementedException extends RuntimeException {
	private static final long serialVersionUID = 3747208892156399108L;

	private static final String message = "501 功能未实现！";

	public NotImplementedException() {
		super(message);
	}

	public NotImplementedException(Throwable throwable) {
		super(message, throwable);
	}
}

File: cloud-collect-core/src/main/kotlin/com/windea/demo/cloudcollect/core/exception/NotImplementedException.kt
package com.windea.demo.cloudcollect.core.exception

import com.windea.demo.cloudcollect.core.domain.enums.*

/**功能未实现的异常。 */
class NotImplementedException : RuntimeException {
	constructor() : super(ResponseResult.NOT_IMPLEMENTED.message)
	
	constructor(throwable: Throwable) : super(ResponseResult.NOT_IMPLEMENTED.message, throwable)
}
